http/src/main/resources/leo.conf

```diff
@@ -65,6 +65,7 @@ proxy {
   # Should match the jupyter wildcard cert
   proxyDomain = ${?PROXY_DOMAIN}
   proxyUrlBase = ${?PROXY_URL_BASE}
+  isProxyCookiePartitioned = ${?IS_PROXY_COOKIE_PARTITIONED}
```

Comment:
Could you link the PR in Terra HelmFile that sets this env variable? I could not find it

Reply:
I think leo.conf overwrites reference.conf